docs/rules/yield-star-spacing.md

```diff
@@ -1,41 +1,39 @@
-# Enforce spacing around the `*` in `yield*` expressions (yield-star-spacing)
+# Enforce spacing around the `*` in `yield*` statements (yield-star-spacing)
 
 (fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
 
 ## Rule Details
 
-This rule enforces spacing around the `*` in `yield*` expressions.
+Enforce a spacing policy for the `*` in `yield*` statements.
```

Comment:
I think we are using "This rule" pretty much everywhere in the first sentence under "Rule Details", so we should Patrick with that. 

Also, please be sure to use complete sentences. The next sentence is missing a subject.